devicetypes/smartthings/aeotec-doorbell-siren-child.src/aeotec-doorbell-siren-child.groovy

```diff
@@ -0,0 +1,57 @@
+/**
+ *	Copyright 2019 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "Aeotec Doorbell Siren Child", namespace: "smartthings", author: "SmartThings", mnmn: "SmartThings", vid: "SmartThings-smartthings-SmartThings_Siren") {
+		capability "Actuator"
+		capability "Health Check"
```

Comment:
This one is using "Health Check" capability, but doesn't define _ping()_ method and it seems that parent device won't take care of it neither.

Reply:
`ping()` is technically optional, but you're right that it should be added if possible